CMakeLists.txt

```diff
@@ -497,12 +497,38 @@ else()
 endif()
 
 if(${PLAYER_AUDIO_BACKEND} MATCHES "^SDL.*$")
-	# speexdsp
-	option(PLAYER_WITH_SPEEXDSP "Resample audio with speexdsp." ON)
-	player_find_package(NAME speexdsp
-		CONDITION PLAYER_WITH_SPEEXDSP
-		DEFINITION HAVE_LIBSPEEXDSP
-		TARGET speexdsp::speexdsp)
+	set(PLAYER_AUDIO_RESAMPLER "Auto" CACHE STRING "Audio resampler to use. Options: Auto speexdsp samplerate OFF")
+	set_property(CACHE PLAYER_AUDIO_RESAMPLER PROPERTY STRINGS Auto speexdsp samplerate OFF)
+
+	if(${PLAYER_AUDIO_RESAMPLER} MATCHES "Auto")
+		option(PLAYER_WITH_SPEEXDSP "Use speexdsp as resampler when available." ON)
```

Comment:
Should these `option`s not be moved up, outside of the condition?

Reply:
They are correctly placed there because they are used to configure the options for the autodetection but I will change all these dependent options to CMAKE_DEPENDENT_OPTIONs. This way they are also properly conditionaly hidden in ccmake and cmake-gui. (try disabling PLAYER_WITH_FREETYPE in these tools, PLAYER_WITH_HARFBUZZ will be appear/dissapear)